pydra/__about__.py

```diff
@@ -55,6 +55,8 @@
     'nipype>=1.1.3',
     'networkx>=%s' % NETWORKX_MIN_VERSION,
     'pytest>=%s' % PYTEST_MIN_VERSION,
+    'dataclasses; python_version < "3.7"',
+    'pyld'
```

Comment:
Some missing imports when running tests locally
```suggestion
    'pyld',
    'cloudpickle',
    'dask',
    'distributed',
```

Reply:
I think `pyld` is only needed for `provenance`, and `dask` and `distributed` shouldn't be required at this stage, unless I've missed something. Perhaps move the non-core dependencies into a `'test'` extra for now, and we can think about splitting them out later?

Reply:
sounds good to me - I realized `dask` and `distributed` are already under `plugins`. we could move `pyld` to `dev`, but we should keep `cloudpickle` in our core

Reply:
at present pyld is only required to collect messages, so it can be moved to testing.